apps/web/src/components/data-table/monitor/data-table-row-actions.tsx

```diff
@@ -85,6 +84,35 @@ export function DataTableRowActions<TData>({
       }
     });
   }
+  async function onClone() {
+    startTransition(async () => {
+      try {
+        const id = monitor.id;
+        if (!id) return;
+
+        const selectedMonitordata = await api.monitor.getMonitorById.query({
+          id,
+        });
+
+        const cloneMonitorData = {
+          ...selectedMonitordata,
+          active: false,
+          id: undefined,
+          updatedAt: undefined,
+          createdAt: undefined,
+        };
+
+        const createdCloneMonitorData = await api.monitor.create.mutate({
+          ...cloneMonitorData,
+        });
+
+        router.push(`./monitors/${createdCloneMonitorData.id}/edit`);
```

Comment:
```suggestion
        router.push(`./monitors/${createdCloneMonitorData.id}/edit`);
        toast.messsage("Monitor cloned!")
```

lets add a toast message to it!